apps/server/src/app/prompts/prompts.resolver.ts

```diff
@@ -238,6 +238,30 @@ export class PromptsResolver {
     return prompt;
   }
 
+  @Mutation(() => Prompt)
+  async deletePrompt(
+    @Args("data") data: PromptWhereUniqueInput,
+    @CurrentUser() user: RequestUser
+  ) {
+    const { id } = data;
+    this.logger.assign({ id }).info("Deleting prompt");
+
+    let prompt: Prompt;
+
+    try {
+      prompt = await this.promptsService.deletePrompt(id);
+    } catch (error) {
+      this.logger.error({ error }, "Error deleting prompt");
+      throw new InternalServerErrorException();
+    }
+
+    this.analytics.track("PROMPT:DELETED", user.id, {
+      promptId: id,
+    });
+
```

Comment:
Please add validation. The user must be an org member to delete a prompt.

We have a utility function for this, called `isOrgMemberOrThrow(user, orgId)`.